vivisect/renderers/__init__.py

```diff
@@ -72,7 +72,7 @@ def renderLocation(self, mcanv, loc, name, isfunc, cmnt, extra):
             segva, segsize, segname, segfname = seg
             if segva == lva:
                 mcanv.addText(linepre, tag=vatag)
-                mcanv.addText("Segment: %s (%d bytes) FIXME PERMS\n" % (segname, segsize))
```

Comment:
We should probably just do this instead of deleting the note to do it.

Reply:
do you know what the FIXME means?  i don't.  meanwhile, it just plain looks bad, and has for 15+ years.  

i cut it because it's been there so long that the context is lost.  Segments don't hold permissions.  perhaps the FIXME means Visi intended to store permissions in the segments, which is a relatively large pile of poo, involving specifics of PE and ELF and MACHO, and modification to the event stream/handler.  i'm pretty sure it was never implemented because it was never important enough.  

i removed it because i don't know how to "FIXME" and i intend for Viv's audience to get much bigger in the near future, and i don't want their first impressions to be shaped by this message.  we can add it to the backlog if you like, but in the UI is just wrong.